includes/events/class-omise-event-charge-capture.php

```diff
@@ -78,7 +78,10 @@ public function resolve() {
 				if ( ! $this->order->has_status( 'processing' ) ) {
 					$this->order->update_status( 'processing' );
 				}
-			break;
+				break;
+
+			default:
```

Comment:
should we raise an exception if we ever reach this? since something would have gone wrong right? 🤔